include/af/blas.h

```diff
@@ -158,8 +158,6 @@ namespace af
         \note optLhs = AF_MAT_CONJ and optRhs = AF_MAT_NONE will run conjugate dot operation.
         \note This function is not supported in GFOR
 
-        \returns out = dot(lhs, rhs)
```

Comment:
It would be nice to replace this with a sentence

\returns dot product of lhs and rhs

Reply:
There is a sentence already, what I removed is duplicate statement